### PR TITLE
Enhancements to Spoof

### PR DESCRIPTION
**Pull request by brophdawg11**
Some Vue cleanup and some helpful abstractions for dealing with `chrome.storage.sync` and `console` logging

### CODE REVIEW

File: app/manifest.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "webRequestBlocking",
     "background",
     "storage",
+    "activeTab",
```
Comment (brophdawg11) [inline app/manifest.json]: Allows us to log console messages to the active tab

File: app/scripts.babel/background.js [hunk 2]
```diff
@@ -1,56 +1,71 @@
...
+  // Don't modify headers on these requests
+  const blacklist = [
+    /\/static\/lib\//,
+    /\/static\/[0-9]\.[0-9]\.[0-9]\/.*\.js/,
+    /\/static\/[0-9]\.[0-9]\.[0-9]\/.*\.css/,
+    /\/static\/[0-9]\.[0-9]\.[0-9]\/images\//,
+    /images\.anthropologie.com/,
+    /images\.urbanoutfitters.com/,
+    /images\.freepeople.com/
+  ];
```
Comment (brophdawg11) [inline app/scripts.babel/background.js]: Skip modifications on certain URL patterns

**Comment by hunt3r** [comment]
This is all great, but I think you should assign to people who are more familiar with the vue syntax than I am.

**Comment by brophdawg11** [comment]
Haha, fair enough - I figured you were the only one who had any knowledge of this tool and how it worked.  I'll find someone with some Vue knowledge and sit down with them to walk through how to run and test it locally

**Pull request merged**